### DISCUSSION TITLE
Welding tool / Plasma cutter combat priority at 10, bots keep using them as weapons

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14271] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Bots keep using their welding tools and plasma cutters as weapons if an enemy is nearby instead of wrench/screwdrivers.

Combat priority for welder/cutter is at 10, combat priority for wrench/screwdriver is at 8/7

Welding tools deal very little damage and waste fuel, while mechanic and engineer respectively have talents to hit harder with their respective tools

### Reproduction steps

spawn a monster near mechanic/engineer bots

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested, no issues found. Closing discussion.
